app/controllers/sessions_controller.rb

```diff
@@ -20,7 +21,19 @@ def create
 
   def destroy
     authorize :session
-    session[:user_id] = nil
+    session.delete(:user_id)
+    cookies.delete(:_feed_reader_user_id)
     redirect_to new_session_path
   end
+
+  private
+
+  def set_session_cookie
```

Comment:
```suggestion
  def set_remember_cookie
```
Or `long_term`, or something else entirely, but this is a confusing name, IMO.